sensor_msgs/NavSatFix goal_latlong
---
geometry_msgs/PoseStamped goal_xy